src/main/java/io/github/nucleuspowered/nucleus/modules/chat/config/ChatConfig.java

```diff
@@ -6,21 +6,34 @@
 
 import ninja.leaping.configurate.objectmapping.Setting;
 import ninja.leaping.configurate.objectmapping.serialize.ConfigSerializable;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.service.permission.Subject;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 @ConfigSerializable
 public class ChatConfig {
 
     @Setting(value = "modifychat", comment = "loc:config.chat.modify")
     private boolean modifychat = true;
 
-    @Setting("template")
-    private ChatTemplateConfig template = new ChatTemplateConfig();
+    @Setting("templates")
+    private Map<String, ChatTemplateConfig> template = new HashMap<String, ChatTemplateConfig>() {{
+        put("Default", new ChatTemplateConfig());
```

Comment:
If the default section was deleted, what would happen?

I'd be inclined to keep the old "template" section as the default, and have this section (maybe `group-templates`) with an example for "default" in.